package.json

```diff
@@ -1,47 +1,77 @@
 {
   "name": "ember-cli-webcomponents-bundler",
   "version": "0.0.0",
-  "description": "The default blueprint for ember-cli addons.",
+  "description": "Allows to use Web Components using ES modules in your Ember app",
   "keywords": [
-    "ember-addon"
+    "ember-addon",
+    "web components",
+    "ES6 modules",
+    "bundle"
   ],
-  "repository": "",
+  "repository": "https://github.com/BBVAEngineering/ember-cli-webcomponents-bundler.git",
   "license": "MIT",
-  "author": "",
+  "author": "BBVAEngineering",
   "directories": {
-    "doc": "doc",
     "test": "tests"
   },
+  "files": [
+    "index.js",
+    "config/babel.js",
+    "config/output.js",
+    "config/rollup.js",
+    "utils/**"
+  ],
```

Comment:
Remove all unwanted files.

Reply:
I'm using `files` field instead of `.npmignore`. It's easier to include only what you need in the package that to exclude a lot of config files and edit the `.npmignore` everytime you add a new unwanted file. You can check that the distributed package will be ok by executing `npm pack` inside the addon.

Reply:
I meant to remove `addon`, `app`, etc... folders from the project. You don't use them.

Reply:
Ok. I removed some of them but not all...

Reply:
Removed in f391834